### PR TITLE
Fix types inconsistencies

### PR DESCRIPTION
**Pull request by brunoscopelliti**
Fix type Account

Extend getOrders/getFills signature to accept props beyond pageargs

(see #231)

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -78,29 +78,27 @@ declare module 'gdax' {
     }
 
     export type PageArgs = {
```
Comment (fb55) [inline index.d.ts]: One of these is required — shouldn't that be enforced?

Comment (brunoscopelliti) [reply index.d.ts]: Currently, I am using `getOrders` and `getFills` without specifying any of those parameter, and the result I get back is reasonably correct. I am not sure those parameters are mandatory; am I wrong?

Comment (fb55) [reply index.d.ts]: Right, they are only required if you want to access another page :D

**Comment by fb55** [comment]
Awesome, thanks a lot!

**Comment by brunoscopelliti** [comment]
Hey @fb55 ,
Do you have any plan about publishing a new package on npm, containing this fix?

**Pull request merged**